ledger/src/shred.rs

```diff
@@ -211,6 +214,15 @@ pub struct Shred {
     pub payload: Vec<u8>,
 }
 
+#[derive(Debug, Default, Clone)]
+pub struct Shreds {
+    pub shreds: Vec<Shred>,
```

Comment:
nit: `shreds -> inner_shreds` so we can avoid calls that look like `shreds.shreds`